app/models/vm_or_template/operations.rb

```diff
@@ -118,6 +118,16 @@ def validate_vm_control
       msg ||= _("Provider doesn't support vm_destroy") unless ext_management_system.respond_to?(:vm_destroy)
       unsupported_reason_add(:terminate, msg) if msg
     end
+
+    supports :suspend do
+      msg = unsupported_reason(:control) unless supports_control?
+      msg ||= _("The VM is not powered on") unless vm_powered_on?
+      unsupported_reason_add(:suspend, msg) if msg
+    end
+  end
+
+  def vm_powered_on?
+    current_state == "on"
```

Comment:
please add the commit with the failing spec to that PR so we can continue the discussion on that code.
It might be valid to change the specs (they are sometimes a bit, well, off)
And yes, it seems that `ManageIQ::Providers::Redhat::InfraManager` does not support suspending of vms.